data/mods/Xedra_Evolved/skills.json

```
@@ -2,7 +2,7 @@
   {
     "type": "skill",
     "ident": "deduction",
-    "name": { "str": "Investigation" },
+    "name": { "str": "investigation" },
```

```suggestion
    "name": { "str": "deduction" },
```
Let's just go ahead and make the change based on the discussion below.